esp/services/ws_fs/ws_fsBinding.cpp

```diff
@@ -370,6 +300,54 @@ IPropertyTree* CFileSpraySoapBindingEx::createPTreeForXslt(const char* method, c
     return pRoot.getClear();
 }
 
+//For every ECLWatchVisible dropzones, read: dropZoneName, directory, and, if available, computer.
+//For every Servers/Server in ECLWatchVisible dropzones, read: directory,
+// server name, and hostname(or IP). Create dropzone("@name", "@directory", "@computer",
+// "@netAddress", "@linux", "@sourceNode") tree into pSoftware.
+void CFileSpraySoapBindingEx::appendDropZones(double clientVersion, IConstEnvironment* env, const char* dfuwuidSourcePartIP, IPropertyTree* softwareTree)
+{
+    IArrayOf<IConstTpDropZone> list;
+    CTpWrapper dummy;
+    dummy.getTpDropZones(clientVersion, nullptr, true, list);
```

Comment:
I'm not even sure it's valid to take a path that *says* it's a on linux DZ but *looks* like a windows file path and convert it to a linux path.

e.g. blah\a\b\c -> blah/a/b/c

how do you know it's not linux filename blah\a\b\c ?

Is that the kind conversion being done?